assets/terraform-modules/bootkube/assets.tf

```diff
@@ -110,55 +110,50 @@ locals {
 }
 
 # Render kubelet.yaml for kubelet chart
-data "template_file" "kubelet" {
+data "ct_config" "kubelet" {
```

Comment:
`ct_config` stand for "config transpiler" and is related to Flatcar configuration. It has nothing to do with rendering. We should render to `locals` block instead.